public/static/js/spawned_hosts.js

```diff
@@ -61,12 +61,16 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope','$window', '$timeout', 'mciSp
                 if(+new Date(host.expiration_time) > +new Date("0001-01-01T00:00:00Z")){
                   if (host.no_expiration) {
                     host.expires_in = "never"
+                    host.date_for_expiration = new Date();
+                    host.time_for_expiration = new Date();
                   } else {
                     var expiretime = moment().diff(host.expiration_time, 'seconds');
                     host.expires_in = moment.duration(expiretime, 'seconds').humanize();
+                    host.date_for_expiration = new Date(host.expiration_time);
+                    host.time_for_expiration = new Date(host.expiration_time);
                   }
-                  host.date_for_expiration = new Date(host.expiration_time);
-                  host.time_for_expiration = new Date(host.expiration_time);
+                  host.empty_time = new Date("0001-01-01T00:00:00Z")
```

Comment:
Where is this used?

Reply:
good catch, I was using this before I realized you could pass in nil to the time/date picker.